api/src/MemberMatch/Member.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RaceResults.MemberMatch
+{
+    public class Member
+    {
+        public List<string> FirstList;
```

Comment:
For simplicity, we can make these get-only properties (e.g. `public List<string> FirstList { get; }`) and have a constructor that sets them

Reply:
This will be fixed in the next pull request.